newlib/libc/machine/riscv/memcpy.c

```diff
@@ -10,83 +11,139 @@
 */
 
 #if defined(PREFER_SIZE_OVER_SPEED) || defined(__OPTIMIZE_SIZE__)
-//memcpy defined in memcpy-asm.S
+// memcpy defined in memcpy-asm.S
 #else
 
-#include <string.h>
-#include <stdint.h>
 #include "../../string/local.h"
+#include "xlenint.h"
+#include <stdint.h>
+#include <stdio.h>
```

Comment:
what is this used for?

Reply:
oh... that shouldn't have been there. I was using it for debugging